<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogQImageGrabberSettings</class>
 <widget class="QDialog" name="DialogQImageGrabberSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="sourceLabel">
     <property name="text">
      <string>Source</string>
     </property>
     <property name="buddy">
      <cstring>lineEditSource</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEditSource"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="fPSLabel">
     <property name="text">
      <string>FPS:</string>
     </property>
     <property name="buddy">
      <cstring>spinBoxFPS</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spinBoxFPS"/>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="checkBoxNoLimit">
     <property name="text">
      <string>Maximum</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QLabel" name="labelError">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>138</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButtonStart">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButtonMore">
     <property name="text">
      <string>Hide details</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBoxProperties">
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutProperties"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
